spark/src/main/scala/ai/chronon/spark/TableUtils.scala

```diff
@@ -47,7 +47,7 @@ case class TableUtils(sparkSession: SparkSession) {
   private val ARCHIVE_TIMESTAMP_FORMAT = "yyyyMMddHHmmss"
   @transient private lazy val archiveTimestampFormatter = DateTimeFormatter
     .ofPattern(ARCHIVE_TIMESTAMP_FORMAT)
-    .withZone(ZoneId.systemDefault())
+    .withZone(ZoneId.of("UTC"))
```

Comment:
forcing this to use UTC, since systemDefaults can be different between local, CI and production environments; we should be consistent.

Reply:
@hzding621 - PTAL and let me know if you think this might cause issues.